packages/react-native-host/ReactNativeHost.podspec

```
@@ -10,13 +10,24 @@ repo_dir = repository['directory']
 
 new_arch_enabled = ENV['RCT_NEW_ARCH_ENABLED'] == '1'
 preprocessor_definitions = [
-  '$(inherit)',
+  '$(inherited)',
   "USE_HERMES=#{ENV['USE_HERMES'] || '0'}",
+  "USE_THIRD_PARTY_JSC=#{ENV['USE_THIRD_PARTY_JSC'] || '0'}",
+  "USE_V8=#{ENV['USE_V8'] || '0'}",
+]
+cxx_flags = [
+  '$(inherited)',
+  "-DUSE_HERMES=#{ENV['USE_HERMES'] || '0'}",
+  "-DUSE_THIRD_PARTY_JSC=#{ENV['USE_THIRD_PARTY_JSC'] || '0'}",
+  "-DUSE_V8=#{ENV['USE_V8'] || '0'}",
 ]
 if new_arch_enabled
   preprocessor_definitions << 'RCT_NEW_ARCH_ENABLED=1'
   preprocessor_definitions << 'USE_FABRIC=1'
   preprocessor_definitions << 'USE_BRIDGELESS=1' if ENV['USE_BRIDGELESS'] == '1'
+  cxx_flags << '-DRCT_NEW_ARCH_ENABLED=1'
+  cxx_flags << 'USE_FABRIC=1'
+  cxx_flags << 'USE_BRIDGELESS=1' if ENV['USE_BRIDGELESS'] == '1'
```

Missing `-D` here:
```suggestion
  cxx_flags << '-DRCT_NEW_ARCH_ENABLED=1'
  cxx_flags << '-DUSE_FABRIC=1'
  cxx_flags << '-DUSE_BRIDGELESS=1' if ENV['USE_BRIDGELESS'] == '1'
```
Or use `CPP_PREPROCESSOR_DEFINITIONS` instead?